docker/dbb_buffmngrs_handoff/buffmngr.yml

```diff
@@ -0,0 +1,29 @@
+# Handoff location (i.e. where the files are written to).
+handoff:
+  buffer: /data/buffer
+  holding: /data/holding
+
+# Ingest service location (i.e. where the files will be transferred to)
+endpoint:
+  user: mxk
```

Comment:
Is it typical github practice to leave real usernames and machine names in file?

Reply:
No, it's an omission on my side. I'll change it.